packages/app-project/pages/projects/[owner]/[project]/classify/index.js

```diff
@@ -3,5 +3,12 @@ export { default } from '@screens/ClassifyPage'
 
 export async function getServerSideProps({ params, query, req, res }) {
   const { props } = await getDefaultPageProps({ params, query, req, res })
+  const { env } = query
+  if (props.workflowID) {
+    const redirect = env ? `workflow/${props.workflowID}?env=${env}` : `workflow/${props.workflowID}`
```

Comment:
That's weird. Relative URLs are valid in the [Location header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location).

I believe Next 10 (we're on 9.5) fixed some bugs where trailing slashes aren't automatically added to request URLs. This might be one of them. 

I had no problems testing in a local production build of the project app, but we've seen problems before where the Next 9 router behaves differently in dev and production. I'll double check that I have Next 9.5, not 9.4, and test again.

Reply:
The workflow should be appended to the request URL. I think Next is broken.
>A client request for
`https://www.example.com/blog/latest`
may get a server response with a path that is relative because it doesn't start with a slash:
>
>```
>HTTP/1.1 302 Found
>Location: 2020/zoo
>```
>The client removes the path segment after the last slash of the original URL and appends the relative path resulting in
`https://www.example.com/blog/2020/zoo`.

https://en.wikipedia.org/wiki/HTTP_location#Relative_URL_relative_path_example